### DISCUSSION TITLE
Static field visibility declaretion

### DISCUSSION THREAD

**Discussion by workgroupengineering** [AvaloniaUI/Avalonia#8948] [Q&A] [answered]
https://github.com/AvaloniaUI/Avalonia/blob/b2af786ae618ace2a9ac4d3b39dc8b6aa2f82d80/src/Avalonia.Base/Media/TextTrimming.cs#L11

https://github.com/AvaloniaUI/Avalonia/blob/b2af786ae618ace2a9ac4d3b39dc8b6aa2f82d80/src/Avalonia.Base/Media/UnicodeRange.cs#L12
They are correct?

**Comment by kekekeks** [comment, marked as answer]
I believe UnicodeRange.Default should be read-only or a read-only property. s_defaultEllipsisChar should be private

**Comment by Gillibald** [comment]
Both should be readonly y
